panther_driver/src/panther_kinematics.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/python3
+
+import math
+from numpy import clip
+from abc import abstractmethod
+
+import rospy
+
+from geometry_msgs.msg import Twist
+
+
+class PantherKinematics:
+    def __init__(self) -> None:
+        self.robot_width = 0.0
+        self.robot_length = 0.0
+        self.wheel_radius = 0.0  # Distance of the wheel center, to the roller center
+        self.encoder_resolution = None
+        self.gear_ratio = None
+        self.power_factor = None
+        self._wheels_enc_speed = [0.0, 0.0, 0.0, 0.0]
+        self.cmd_vel_command_time = rospy.Time.now()
```

Comment:
Those are public members of class. If access is needed outside of the class, corresponding getters should be defined

Reply:
:+1: